src/hostManager/hostManage.test.ts

```diff
@@ -1,10 +1,21 @@
+import { v4 } from 'uuid';
 import { describe, expect, it } from 'vitest';
 
 import { ClientHostManager } from './hostManager.js';
+import { Context } from '@/enums/context.js';
 
 // TODO: add more tests
```

Comment:
You mean in another PR ?

Reply:
I mean: you left it because I believe you created this file at the very beginning 😆

Reply:
Ah ok oups ahahah my bad 😓